### PR TITLE
NG-exporter changes for better data porting to RalphNG.

### PR DESCRIPTION
**Pull request by ghost**
Mainly resources update.

### CODE REVIEW

File: src/ralph/export_to_ng/management/commands/export_model.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         models = args
         if not models:
-            models = SIMPLE_MODELS + DEPENDENT_MODELS + MANY_TO_MANY
+            models = SIMPLE_MODELS + DEPENDENT_MODELS
```
Comment (mkurek) [inline src/ralph/export_to_ng/management/commands/export_model.py]: what about many-to-many?

File: src/ralph/export_to_ng/resources.py [hunk 2]
```diff
@@ -64,14 +74,20 @@ def dehydrate_rack(self, asset):
...
+            rack = ''
         return rack
 
     def dehydrate_service_env(self, asset):
-        device_environment = (
-            asset.device_environment.name if asset.device_environment else ''
+        service = getattr(
+            asset.service,
+            'name',
+            settings.NG_EXPORTER['default_service_name'],
```
Comment (mkurek) [inline src/ralph/export_to_ng/resources.py]: service could be null now so we shouldn't assign default values here

File: src/ralph/export_to_ng/resources.py [hunk 3]
```diff
@@ -64,14 +74,20 @@ def dehydrate_rack(self, asset):
...
+        )
+        device_environment = getattr(
+            asset.device_environment,
+            'name',
+            settings.NG_EXPORTER['default_environment_name'],
```
Comment (mkurek) [inline src/ralph/export_to_ng/resources.py]: ditto

**Comment by mkurek** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**